docs/en/operations/settings/settings.md

```diff
@@ -4643,3 +4643,19 @@ SELECT toFloat64('1.7091'), toFloat64('1.5008753E7') SETTINGS precise_float_pars
 │              1.7091 │                 15008753 │
 └─────────────────────┴──────────────────────────┘
 ```
+
+## validate_tcp_client_information {#validate-tcp-client-information}
+
+Determines whether validation of client information enabled when query packet is received from a client using a TCP connection.
+
+If `true`, an exception will be thrown on invalid client information from the TCP client. clickhouse-clients using version 23.7 and older are not compatible with the newer 23.8+ server version.
```

Comment:
Let's make them compatible. Currently, you can use `clickhouse-client` version 18.1 to connect to `clickhouse-server` version 23.8 and vice versa. Let's keep it this way. We can do it by allowing both `ClickHouse` and `ClickHouse client` and converting one to another for logging purposes (as discussed in https://github.com/ClickHouse/ClickHouse/pull/53673)

Reply:
done